### ISSUE TITLE
Warning: Cannot find module '/app/out/node_modules/grunt-usemin/lib/config/uglifyjs' Use --force to continue.

### ISSUE THREAD

**Issue by GuGuss** [yeoman/grunt-usemin#590]
Since tagging the 3.1 version today, I got the following issue when running ``grunt build``: 

````
Warning: Cannot find module '/app/out/node_modules/grunt-usemin/lib/config/uglifyjs' Use --force to continue.
````

I guess [this commit](https://github.com/yeoman/grunt-usemin/commit/748261430807582280ab99d4b12760dd1646b321) should be considered as a breaking change.

**Comment by sergio773** [comment]
This issue is duplicated here: https://github.com/yeoman/grunt-usemin/issues/589

**Comment by GuGuss** [comment]
OK.

**Comment by stephanebachelier** [comment]
This was needed to fix the issue in #409. So yes a breaking change since 3.0
